app/src/main/resources/application.properties

```diff
@@ -9,11 +9,13 @@ quarkus.datasource.db-kind=h2
 quarkus.hibernate-orm.database.generation=drop-and-create
 
 # vault url
-%prod.quarkus.vault.url=http://localhost:8200
+%prod.quarkus.vault.url=http://vault.65.108.212.158.nip.io
+%dev.quarkus.vault.url=http://vault.127.0.0.1.nip.io
 
 # vault authentication
-%prod.quarkus.vault.authentication.userpass.username=bob
-%prod.quarkus.vault.authentication.userpass.password=sinclair
+quarkus.vault.authentication.userpass.username=bob
+quarkus.vault.authentication.userpass.password=sinclair
+
+%prod.quarkus.vault.kv-secret-engine-version=1
```

Comment:
Why do you use as version kv v1 while we agreed to use kv v2 ? @aureamunoz

Reply:
Because this was using vault kv secret engine v1 installed by `vault.sh`.